updater/src/Updater/Generate/Changelog.purs

```diff
@@ -100,9 +102,12 @@ fetchNextPageOfReleases page gh = do
   case result of
     Left err -> do
       throwError (error $ AX.printError err)
-    Right { body } ->
+    Right { status } | status == (StatusCode 404) -> do
+      pure Nothing
```

Comment:
[Q] Should we just ignore it or throw an error?

Reply:
See my other comment.